collect_app/src/main/AndroidManifest.xml

```diff
@@ -176,9 +176,8 @@ the specific language governing permissions and limitations under the License.
 
         <activity android:name=".activities.FormMapActivity" />
         <activity android:name=".activities.BearingActivity" />
-        <activity android:name=".activities.CaptureSelfieActivity" />
+        <activity android:name="org.odk.collect.selfiecamera.CaptureSelfieActivity" />
```

Comment:
Just to me use I understand correctly... Do we have to register it here or we could do that in the module's manifest file?
In what case we have to use module's manifest file? When such activities are started from that module not from the app one?

Reply:
Android Library manifests are merged with the app one, so it actually doesn't matter where it lives. It would need to be in the module's manifest if there was a test that starts `CaptureSelfieActivity` and in that case it could then be removed from the app's manifest.

Reply:
So shouldn't we do that in the module's manifest? Modules should be separate units so someone should be able to take this module and add to another project without adding anything else, but in this case registering that activity in the manifest file would be required. I know there are other dependencies too and our modules are not perfect but if we can do something in that direction maybe we should.

Reply:
> Modules should be separate units so someone should be able to take this module and add to another project without adding anything els

Yeah that's a good argument for defaulting to having it in the module. I'll move it!